docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Recommendation/OneClassMatrixFactorizationWithOptions.cs

```diff
@@ -9,60 +9,80 @@ namespace Samples.Dynamic.Trainers.Recommendation
+        // In this example we will create in-memory data and then use it to train a
+        // one-class matrix factorization model. Afterward, prediction values are
+        // reported. To run this example, it requires installation of additional
+        // nuget package Microsoft.ML.Recommender found at
+        // https://www.nuget.org/packages/Microsoft.ML.Recommender/
         public static void Example()
         {
-            // Create a new context for ML.NET operations. It can be used for exception tracking and logging,
-            // as a catalog of available operations and as the source of randomness.
+            // Create a new context for ML.NET operations. It can be used for
+            // exception tracking and logging, as a catalog of available operations
+            // and as the source of randomness.
             var mlContext = new MLContext(seed: 0);
 
             // Get a small in-memory dataset.
-            GetOneClassMatrix(out List<MatrixElement> data, out List<MatrixElement> testData);
+            GetOneClassMatrix(out List<MatrixElement> data,
+                out List<MatrixElement> testData);
 
-            // Convert the in-memory matrix into an IDataView so that ML.NET components can consume it.
+            // Convert the in-memory matrix into an IDataView so that ML.NET
+            // components can consume it.
             var dataView = mlContext.Data.LoadFromEnumerable(data);
 
-            // Create a matrix factorization trainer which takes "Value" as the training label, "MatrixColumnIndex" as the
-            // matrix's column index, and "MatrixRowIndex" as the matrix's row index. Here nameof(...) is used to extract field
+            // Create a matrix factorization trainer which takes "Value" as the
+            // training label, "MatrixColumnIndex" as the matrix's column index, and
+            // "MatrixRowIndex" as the matrix's row index. Here nameof(...) is used
+            // to extract field
             // names' in MatrixElement class.
             var options = new MatrixFactorizationTrainer.Options
             {
-                MatrixColumnIndexColumnName = nameof(MatrixElement.MatrixColumnIndex),
-                MatrixRowIndexColumnName = nameof(MatrixElement.MatrixRowIndex),
-                LabelColumnName = nameof(MatrixElement.Value),
-                NumberOfIterations = 20,
-                NumberOfThreads = 8,
-                ApproximationRank = 32,
-                Alpha = 1,
-                // The desired values of matrix elements not specified in the training set.
-                // If the training set doesn't tell the value at the u-th row and v-th column,
-                // its desired value would be set 0.15. In other words, this parameter determines
-                // the value of all missing matrix elements.
+                MatrixColumnIndexColumnName = nameof(
+                    MatrixElement.MatrixColumnIndex),
```

Comment:
Line 46 and Lines 47-53 should be at the same indent level.

Reply:
Zeeshan asked us to indent after the first break, so we've been doing that for all the files. Maybe we can discuss this in the scrum tomorrow?